From 3f1c9a7e5b8d4c2a9e6f0d1b7a4c8e2f5d9b3a61 Mon Sep 17 00:00:00 2001
From: Gregory Norton <gregory@serenade.local>
Date: Sat, 29 Aug 2026 00:00:00 +0000
Subject: [PATCH 1/2] LibGUI: Include <limits.h> for INT_MIN

Window.cpp uses INT_MIN but relied on a transitive include from the
Serenity LibC headers, which does not exist when building against a
host libc (Lagom). Include what we use.

---
 Userland/Libraries/LibGUI/Window.cpp | 1 +
 1 file changed, 1 insertion(+)

diff --git a/Userland/Libraries/LibGUI/Window.cpp b/Userland/Libraries/LibGUI/Window.cpp
index 8a60f3d076..cf9453f369 100644
--- a/Userland/Libraries/LibGUI/Window.cpp
+++ b/Userland/Libraries/LibGUI/Window.cpp
@@ -29,6 +29,7 @@
 #include <LibGfx/Bitmap.h>
 #include <LibGfx/Palette.h>
 #include <fcntl.h>
+#include <limits.h>
 #include <stdio.h>
 #include <stdlib.h>
 #include <unistd.h>
-- 
2.40.0
